docs/docs/features/low-power-states/_sideband-wakeup-matrix.md

```diff
@@ -0,0 +1,11 @@
+The kscan sideband behavior needs to wrap the kscan due to technical reasons:
+
+```
+/ {
+    side_band_behavior_triggers: side_band_behavior_triggers {
+        compatible = "zmk,kscan-sideband-behaviors";
+        wakeup-source;
+        kscan = <&kscan>;
```

Comment:
Double checking: does this not need `auto-enable`?

Reply:
Neither approach needs `auto-enable` prior to [048cb0](https://github.com/zmkfirmware/zmk/commit/0438cb0ee532a31291aee3dabe24362f16146fd2). Since matrix integrated solutions have been broken since then (#2376), I'm not sure how to answer that. Pete's changes in said commit didn't add `auto-enable` to the docs for the matrix case, so I haven't included it either.

Reply:
I guess we'll ask Pete about it when he can get around to it.